app/controllers/actions/IdentifierAction.scala

```diff
@@ -53,12 +53,20 @@ class AuthenticatedIdentifierAction @Inject()(override val authConnector: AuthCo
           block(AdministratorRequest(affGroup, internalId, psaId.value, UserType.PSA, request))
         case Some(internalId) ~ Some(affGroup) ~ IsPSP(pspId) if hasValidSession(hc) =>
           block(PractitionerRequest(affGroup, internalId, pspId.value, UserType.PSP, request))
-        case Some(_) ~ Some(_) ~ _ => Future.successful(Redirect(config.loginUrl))
-        case _ => Future.successful(Redirect(routes.UnauthorisedController.onPageLoad()))
+        case Some(_) ~ Some(_) ~ _ if !hasValidSession(hc) =>
+          Future.successful(Redirect(controllers.auth.routes.AuthController.sessionTimeout()))
+        case Some(_) ~ Some(_) ~ _ =>
+          throw InsufficientEnrolments("No sufficient enrolments found")
```

Comment:
I don't think this is technically possible since if they don't have either enrolment the initial auth call will completely fail